ystia/yorc-a4c-plugin
# add exception throw when not able to retrieve yorc logs

# Pull Request description

## Description of the change

### What I did
Throw errors

### How I did it
I added an exception throw in restClient code

### How to verify it

- Run an alien (docker or not) 
- Launch Yorc 

- [ ] Try to connect to a wrong yorc IP and verify it does not work and display an error
- [ ] Connect to a good yorc IP and verify it works and display "connected"


### Description for the changelog

## Applicable Issues

Resolves https://github.com/ystia/yorc-a4c-plugin/issues/72
